addons/test_mail/tests/test_performance.py

```diff
@@ -995,7 +995,7 @@ def test_message_post_template(self):
         template = self.env.ref('test_mail.mail_test_container_tpl')
 
         # about 20 (19 ?) queries per additional customer group
-        with self.assertQueryCount(admin=65, employee=65):
+        with self.assertQueryCount(admin=66, employee=66):
```

Comment:
Probably due to the call on `object._mail_get_customer().lang` when there is no `lang` set for the given template?